e2e/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
What is this used for? Can we remove?

Reply:
Fixtures file is also something that the Cypress test running needs in order to run. If we remove it similar to the config file it will be added back again.

Reply:
oh that's surprising, since the test doesn't use these fixtures. I'd imagine having an empty folder called fixtures would be alright - That's alright though

Reply:
I removed the file and left the folder - Since if it does need it, it will just add it back whenever the tests run.